ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-stable-4.17-upgrade-from-stable-4.16.yaml

```diff
@@ -59,6 +59,23 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
+- as: aws-ipi-disc-priv-arm-mixarch-f28
+  cron: 21 12 8 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      ADDITIONAL_WORKER_ARCHITECTURE: x86_64
+      ADDITIONAL_WORKER_VM_TYPE: m6i.xlarge
+      ADDITIONAL_WORKERS: "1"
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_REPLICAS: "2"
+      COMPUTE_NODE_TYPE: m6g.xlarge
+      CONTROL_PLANE_INSTANCE_TYPE: m6g.xlarge
+      EXTRACT_MANIFEST_INCLUDED: "true"
+      OCP_ARCH: arm64
+    test:
+    - chain: openshift-upgrade-qe-sanity-heterogeneous
```

Comment:
I guess you copy this job from openshift-openshift-tests-private-release-4.16__multi-stable.yaml. If I remember right, the chain is changing the cluster to heterogeneous cluster after upgrade, but not updating a heterogeneous cluster, while the job in openshift-openshift-tests-private-release-4.16__multi-stable.yaml is installing a heterogeneous cluster before upgrade.

Reply:
Replaced with:
```
    - ref: ipi-install-heterogeneous
    - chain: openshift-upgrade-qe-sanity
```